code/modules/ntsl2/ntsl2.dm

```diff
@@ -0,0 +1,46 @@
+/NTSL_interpreter
+	var/connected = 0
+	var/list/programs = list()
+
+	New()
```

Comment:
```suggestion
/datum/NTSL_interpreter/New()
```
Please don't use relative pathing, and AFAIK things are a subtype of datum by default so please make it explicit.